modules/mono/editor/Godot.NET.Sdk/Godot.NET.Sdk/Sdk/Sdk.props

```diff
@@ -2,8 +2,6 @@
   <PropertyGroup>
     <!-- Determines if we should import Microsoft.NET.Sdk, if it wasn't already imported. -->
     <GodotSdkImportsMicrosoftNetSdk Condition=" '$(UsingMicrosoftNETSdk)' != 'true' ">true</GodotSdkImportsMicrosoftNetSdk>
-
-    <GodotProjectTypeGuid>{8F3E2DF0-C35C-4265-82FC-BEA011F4A7ED}</GodotProjectTypeGuid>
```

Comment:
Hey! Why have you removed GodotProjectTypeGuid?
Rider UnitTesting feature uses it for special casing Godot project.
https://github.com/JetBrains/godot-support/blob/f6d1349dfad81fc127a53e7c0671d95c6752770c/resharper/src/UnitTesting/GodotRunHostProvider.cs#L50

Reply:
I was able to fix it by adding ProjectTypeGuids manually.
https://github.com/van800/godot-demo-projects/commit/03594a85f5898645c569f91bebfca6381c34c5e4

Reply:
We used `ProjectTypeGuids` for the Visual Studio project system, but it no longer has any effect after we switched to Sdk style csproj. The property prevents Visual Studio for Mac from opening the project unless the Godot extension is installed, which I consider a problem, so it was removed. Could Rider do something different for special casing Godot projects? If not I'm willing to add it back (and maybe it works `BuildingByReSharper` condition?).

Reply:
I'll see, what I can do, thank you for clarification.